api/src/paths/project/{projectId}/survey/{surveyId}/critters/index.ts

```diff
@@ -163,6 +182,73 @@ POST.apiDoc = {
   }
 };
 
+PATCH.apiDoc = {
```

Comment:
You added this update "PATCH" to the index file. To follow endpoints done in SIMS we have been putting Update and Delete functions in the `_id` file. Example of this is `api\src\paths\project\{projectId}\participants` with the `index.ts` file and `{projectParticipationId}.ts`

Reply:
Agreed, this should be moved as a URL parameter. That is, unless we're deleting several records at once

Reply:
I consciously did not do this to keep it more in line with the usage in Critterbase, where you are actually allowed to update multiple critters at once. But I dunno if it will ever see that usage here, so the argument to structure it that way could make sense.